CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 - In advance search, any status tag is showing archived after search [#6678](https://github.com/opencrvs/opencrvs-core/issues/6678)
 - Fix first name issues when creating a user [#6631](https://github.com/opencrvs/opencrvs-core/issues/6631)
 - Show correct record option in certificate preview page when trying to print by RA [#6224](https://github.com/opencrvs/opencrvs-core/issues/6224)
+- Change condition of Number of previous births [#6356](https://github.com/opencrvs/opencrvs-core/issues/6356)
```

Comment:
Use OpenCRVS core specific language here so for example, from Core's point of view this change would be 

"Allow defining `maxLength` attribute for number type fields"